src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement, useState } from "react";
```

Comment:
there is a similar file named stepper.tsx could you please remove the one not used.

Reply:
We have only one file that is Stepper.tsx. Git was not able to see the case sensitivity. I have configured it now. it will be fine.